---
layout: page
title: ""
subtitle:
full-width: true
---
## Long non-coding RNAs in neurodevelopment  
<div style="text-align: justify">
Long non-coding RNAs (lncRNAs) are a recently defined class of PolII transcripts that are longer than 200nt and lack coding potential. These transcripts are largely believed to exert regulatory pressure on genes with the more famous lncRNAs being ones that modulate developmental genes. As of now, my thesis project revolves around the significance of two lncRNAs in murine neocortical development: Pantr1 and Pantr2. These transcripts flank the Pou3f3 (Brn1) locus, a protein coding gene that works with Pou3f2 (Brn2) to correctly pattern the cortex during development. Knockout of Pantr1 in embryonically derived cortical neurospheres is known to lead to an increase in genes associated with stem cell maintenance and proliferation as determined by bulk RNAseq. It is, however, not known how exactly Pantr1 regulates the expression of these genes. In contrast, knockout of Pantr2 appears to have the opposite effect in cortical neurospheres; upregulating genes involved in differentiation and neuronal maturation. Additionally, the neocortex of Pantr2 null mice show extensive laminar disorganization coupled with a loss of layer IV cortical barrel structures. While interesting, we still are unsure how loss of Pantr2 leads to such extensive cortical defects.
</div>

## Neurodevelopment and disease  
<div style="text-align: justify">
A long term research interest of mine centers around neurodevelopment as it pertains to disease. Neurodevelopmental disorders such as autism spectrum disorders (ASD) are not very well understood, due in part to the lack of understanding neural diversity and development. As such, I'm interested in aiding in untangling the diversity present in key regions in the central nervous system (CNS) such as the cortex and hippocampus. My principle approach in uncovering cell types involved in neuraldevelopmental disorders is single cell RNA sequencing (scRNAseq). scRNAseq is a quickly growing technique used to profile the transcriptomes of thousands of individual cells. From the data collected, we can assign cellular identity and cellular state, as well as cluster cells based on similar gene expression profiles. Currently, this technique is primarily being used to annotate cell atlases for different tissues of interest. While curating cell types within a given tissue is useful, important information about novel cell types can be discerned by analyzing datasets with observable phenotypes.
</div>

<br>
<br>
<br>
<br>
<br>
